### PR TITLE
v0.3

### PR DESCRIPTION
**Pull request by simbleau**
### Added

- Added `velato::Renderer::render`, which now returns a new vello scene.

### Changed

- Updated to vello 0.2
- Renamed `VelatoError` to `velato::Error`
- Renamed the existing `velato::Renderer::render` to `velato::Renderer::append`

### Removed

- All code and related profiling (`wgpu_profiler`) used in examples.

### CODE REVIEW

File: src/error.rs
Comment (DJMcNab) [inline src/error.rs:1]: I'd be inclined to put this in `src/lib.rs`

**Review by DJMcNab — APPROVED** [review]
Standard optimistic approval for Velato

I have not tested this

File: src/lib.rs [hunk 1]
```diff
@@ -3,10 +3,12 @@
 
 //! Render a Lottie animation to a Vello [`Scene`](crate::vello::Scene).
 //!
-//! However, this is also intended to be the preferred integration between Vello and [Lottie](https://lottie.github.io/lottie-spec/), so [consider
-//! contributing](https://github.com/linebender/velato) if you need a feature which is missing.
+//! This currently lacks support for a [number of important](crate#unsupported-features) SVG features.
```
Comment (dominikh) [inline src/lib.rs:6]: How are Velato/Lottie and SVG related?

Comment (simbleau) [reply src/lib.rs:6]: This is a typo, probably since we copied this part from vello_svg.

**Pull request merged**